mayastor-test/test_common.js

```diff
@@ -358,11 +358,7 @@ function dumbCommand (method, args, done) {
       JSON.stringify(args) +
       "'",
     (err, stdout, stderr) => {
-      if (err) {
-        done(new Error(stderr));
-      } else {
-        done();
-      }
+      done(err, stdout);
```

Comment:
The motivation for that change is so that we can use the output to retrieve the actual published URIs from the nexuses. That is what we store in published URIs

Reply:
yes, I got why the stdout is there, but I don't get why you throw stderr away in case of an error.